service/src/main/java/bio/terra/profile/service/iam/SamService.java

```diff
@@ -23,11 +24,7 @@
 import org.broadinstitute.dsde.workbench.client.sam.api.ResourcesApi;
 import org.broadinstitute.dsde.workbench.client.sam.api.StatusApi;
 import org.broadinstitute.dsde.workbench.client.sam.api.UsersApi;
-import org.broadinstitute.dsde.workbench.client.sam.model.AccessPolicyMembershipV2;
-import org.broadinstitute.dsde.workbench.client.sam.model.CreateResourceRequestV2;
-import org.broadinstitute.dsde.workbench.client.sam.model.ResourceAndAccessPolicy;
-import org.broadinstitute.dsde.workbench.client.sam.model.SystemStatus;
-import org.broadinstitute.dsde.workbench.client.sam.model.UserStatusInfo;
+import org.broadinstitute.dsde.workbench.client.sam.model.*;
```

Comment:
I was asked to avoid `*` imports in WSM, do we want to do the same in BPM?

Reply:
Hmmm, I personally think that `*` is fine, especially if there is nothing in place to prevent them from being used. If we are going to enforce this, it think it needs to me a spotless rule.

@aherbst-broad do you have an opinion?

Reply:
I also think `*` is fine. 
The original (as in, in the 1990s) reason to avoid it in Java was to avoid name collisions; this doesn't matter anymore, because Java supports the ability to use explicit imports to clarify those cases.
For example:
```Java
import java.util.*
import project.package.that.contains.something.called.list.*
import java.util.List
```
Static initialization happens when the class is loaded (before `main` is called), so unless you were dynamically loading custom jars or something weird, that wouldn't be a concern.

Reply:
> The original (as in, in the 1990s) reason to avoid it in Java was to avoid name collisions; this doesn't matter anymore, because Java supports the ability to use explicit imports to clarify those cases.

I've been using Java since the 1990s (I worked on one of the first Java IDEs) and I don't think I've ever heard this as a reason to avoid static imports. The reason I've heard (and repeated, because I believe it) is they should be avoided because it makes programs harder to read. This is why I would avoid both static imports and wildcard imports, with the exception being specific classes in test code (hamcrest and mockito APIs).

The [google java style guide](https://google.github.io/styleguide/javaguide.html#s3.3-import-statements), which DSP follows, says `No wildcard imports` and `No static import for classes` (although for static import it sounds like they're specifically talking about nested static classes).

An easy way to enforce this is to use an `.editorconfig` for IntelliJ that has rules to prevent wildcards from being used. Here's the file we use for catalog: https://github.com/DataBiosphere/terra-data-catalog/blob/main/.editorconfig